dep: replace page (#4476)
tldr-pages/tldr: pages/common/dep-deploy.md

> # dep deploy

Reviewer: We named the `md` files with command name
```suggestion
# dep
```

> > A cli tool for deployment of PHP applications.

Reviewer: ```suggestion
> A CLI tool for deployment of PHP applications.
```

> More information: <https://deployer.org>.

> - Initialize deployer in a project, may use a framework recipe template:

Reviewer: I suggest to move the `--template` flag in the example description:
```suggestion
- Interactively initialize deployer in the local path (use a framework template with `--template={{template}}`):
```

> `dep init [-t Template]`

Reviewer: ```suggestion
`dep init`
```

- Deploy an application to remote host:

`dep deploy [{{hostname}}]`

- Rollback to the previous working release:

`dep rollback`

- Connect to host through ssh:

> `dep ssh [{{hostname}}]`

Reviewer: Why the `[]`?
```suggestion
`dep ssh {{hostname}}`
```

- List commands:

`dep list`

- Run any arbitrary command on hosts:

> `dep run [{{host}}]`

Reviewer: The docs has this example:
```
dep run 'ls -la'
```
Why you put `host`?
```suggestion
`dep run "{{command}}"`
```
